server/tserver/src/main/java/org/apache/accumulo/tserver/tablet/Tablet.java

```diff
@@ -1926,7 +1926,7 @@ public void checkIfMinorCompactionNeededForLogs(List<DfsLogger> closedLogs) {
     // grab this outside of tablet lock.
     @SuppressWarnings("deprecation")
     int maxLogs = tableConfiguration.getCount(tableConfiguration
-        .resolve(Property.TSERV_WALOG_MAX_REFERENCED, Property.TABLE_MINC_LOGS_MAX));
+        .resolve(Property.TSERV_WAL_MAX_REFERENCED, Property.TSERV_WALOG_MAX_REFERENCED));
```

Comment:
It looks like `TABLE_MINC_LOGS_MAX` was deprecated in 2.0.0. There should be two resolves here to ensure we use the right one:

```suggestion
        .resolve(Property.TSERV_WAL_MAX_REFERENCED, tableConfiguration.resolve(Property.TSERV_WALOG_MAX_REFERENCED, Property.TABLE_MINC_LOGS_MAX));
```
(this suggestion isn't formatted)

If we have to do this a lot, it might make sense to modify the `resolve` command to take more than one argument, but probably not worth it if there's only one or two.